mz_crypt_openssl.c

```diff
@@ -293,7 +293,7 @@ typedef struct mz_crypt_hmac_s {
 
 /***************************************************************************/
 
-#if (OPENSSL_VERSION_NUMBER < 0x10100000L) || defined(LIBRESSL_VERSION_NUMBER)
+#if (OPENSSL_VERSION_NUMBER < 0x10100000L) || (LIBRESSL_VERSION_NUMBER < 0x2070000fL)
```

Comment:
I think you still need to get check for existence of `LIBRESSL_VERSION_NUMBER`

Reply:
Fair enough, fixed.